### PR TITLE
Use adapter height when drawing a placeholder

### PR DESCRIPTION
**Pull request by planarvoid**
### Fix
This PR fixes an issue when the placeholder size was different (by a little bit) than what was defined in the adapter. This was mostly visible on smaller placeholders. I think only the last placeholder of the document was affected. 

In addition it fixes the width so that the cursor doesn't overlap the placeholder

### Test (it's better to test this with the PR where it's used)
1. Setup placeholders
2. Add 2 placeholders to the `EXAMPLE` 
3. Add logging of actual height to the code
4. Check the height is fixed

### Review
@khaykov 

Make sure strings will be translated:

- [x] If there are new strings that have to be translated, I have added them to the client's `strings.xml` as a part of the integration PR.

### CODE REVIEW

**Review by Teelair — APPROVED** [review]
I tested this using Day One Android on the `fix/too-big-audio-player` branch. 

To test, I made four entries each containing photos, audio files, videos, and a PDF. I took screenshots of the entries looked in both read and edit mode before applying this PR's changes, and then after. 

The most noticeable issue where audio placeholders were visibly larger than others has been fixed, but also other placeholders have had their size slightly altered. It is not by much so I feel like this is not a big deal within the context of Day One, but because I am not sure of what all is using Aztec, I'll leave an approval and leave it up to you to merge this if that sounds okay to you @planarvoid!

:shipit:

**Review by khaykov — APPROVED** [review]
Approving since @Teelair gave a 🟢

**Pull request merged**
